ui/src/graphics.test.tsx

```diff
@@ -32,4 +37,10 @@ describe('Graphics.tsx', () => {
     expect(queryByTestId(name)).toBeInTheDocument()
   })
 
+  it('Renders with background image', () => {
```

Comment:
Let's replace this unit test with an entry in [widgets section](https://wave.h2o.ai/docs/widgets/overview). The best place would probably be Widgets > Plots > Graphics or Widgets > Plots > AI (whatever makes more sense to you) and include 2 variations, a simple graphic and a simple graphic with a background image. That way, our documentation will be better and this functionality will be picked up by our visual regression testing suite.

Feel free to reach out in case you need help with any of the above.

Reply:
Hi @mturoci, question for you about this. Would it be better to add this to Guide > Graphics instead since that page already explains the `ui.graphics_card`? I could add a new section at the bottom to explain the new background image feature. That seems like a more logical place than Widgets > Plots, but maybe I'm overlooking something?

Reply:
Guide > Graphics isn't picked up by visual regression testing suite + is more of a docs on how to work with graphics rather than how to display a widget and all its variations. The widget page should definitely have a link to the graphics guide though.

Reply:
@mturoci, I decided to add the new Graphics section at Widgets > Content > Graphics, just after the section for Image. The graphics stuff didn't seem to fit with the other information under Widgets > Plots, but it seemed to go well with the information under Widgets > Content. Let me know if you disagree and I can move it to wherever you prefer. I built the documentation and here's a short video showing the new page:

https://github.com/user-attachments/assets/c91c2f4d-f501-4f7a-8ca0-3382ec49acdc